index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="wholepage">
+
+        <header id="header">
+            <img id="header-logo"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="">
+
+            <nav id="navbar">
+                <ul>
+                    <li><a href="#featuresSec" class="nav-links">Feature</a></li>
+                    <li><a href="#videoSec" class="nav-links">Video</a></li>
+                    <li><a href="#pricingSec" class="nav-links">Pricing</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <div class="subscription">
+
+            <h2><b><strong>Home-made masterpieces</strong></b></h2>
+
+            <form id="form" action="https://codepen.io/freeCodeCamp/full/RKRbwL">
+                <input id="email" type="email" placeholder="your email...">
+                <input id="input" type="submit" value="Let's Go!">
+            </form>
+        </div>
```

Comment:
Please, kindly change the div tag of each section of your code for the sections tag.
For example:   <section class="subscription">  </section class="subscription">